suite-native/atoms/src/Modal/BottomModal.tsx

```diff
@@ -46,6 +63,60 @@ const modalHeaderStyle = prepareNativeStyle(utils => ({
     paddingVertical: utils.spacings.medium,
 }));
 
+const useAnimationStyles = ({
+    onVisibilityChange,
+}: Pick<BottomModalProps, 'onVisibilityChange'>) => {
+    const translatePanY = useSharedValue(SCREEN_HEIGHT);
+
+    const animatedModalWrapperStyle = useAnimatedStyle(() => ({
+        top: withTiming(interpolate(translatePanY.value, [-1, 0, 1], [0, 0, 1]), {
+            duration: 300,
+            easing: Easing.out(Easing.cubic),
+        }),
+    }));
+
+    const closeModalAnimated = () => {
+        'worklet';
+
+        translatePanY.value = SCREEN_HEIGHT;
+        runOnJS(onVisibilityChange)(false);
+    };
+
+    const resetModalAnimated = useCallback(() => {
+        'worklet';
+
+        translatePanY.value = 0;
+    }, [translatePanY]);
+
+    const panGestureEvent = useAnimatedGestureHandler<
+        PanGestureHandlerGestureEvent,
+        GestureHandlerContext
+    >({
+        onStart: (_, context) => {
+            context.translatePanY = translatePanY.value;
+        },
+        onActive: (event, context) => {
+            const { translationY } = event;
+            translatePanY.value = translationY + context.translatePanY;
+        },
+        onEnd: event => {
+            const { translationY, velocityY } = event;
+            if (translationY > 0 && velocityY > 2) {
```

Comment:
Yeah. Looks fine to me

Reply:
I don't think it's a default sheet behavior: 

1) I would put `translationY` to about 50 to make is more bouncy
2) Even if velocity is small but `translationY` is big, it should close

It still might change in the future but it would be easier to tune it if the baseline is closer to the final! :)